Documentation/Usage.md

```diff
@@ -393,6 +393,49 @@ import Foundation
 #endif
 ```
 
+## Dependency Mirroring
+
+A dependency mirror refers to an alternate source location which exactly replicates the contents of the original source. Mirrors can ensure availability even if original source is moved/unavailable or deleted. Mirror information is stored in a local file at `<PackageRoot>/.swiftpm/config` in JSON format, however there are commands which allow you to add/delete without having to edit the file manually.
```

Comment:
> which **exactly replicates** the contents of the original source

That's not strictly true, as you can fork a package for the purpose of making changes to it. I'd leave out the "exactly replicates" bit. How about:

"A dependency mirror refers to an alternate source location where a package is made available. Mirrors can ensure availability even if the original location is unavailable, or has been moved or deleted. [...]"